cves/kernel/CVE-2020-24394.yml

```diff
@@ -229,8 +228,8 @@ subsystem:
     e.g.
         name: ["subsystemA", "subsystemB"] # ok
         name: subsystemA # also ok
-  name:
-  note:
+  name: fs
```

Comment:
If your vulnerability is about files on a domain, then net is also an applicable subsystem.

Reply:
I added this